website/docs/help.md

```diff
@@ -1,15 +1,74 @@
 ---
 id: help
-title: Want to help?
+title: How to contribute
 ---
+Thanks for your interest in contributing to the Gatekeeper project! This document will help answer common questions you may have during your contribution.
 
+## Where to start?
 Join us to help define the direction and implementation of this project!
 
-- Join the [`#kubernetes-policy`](https://openpolicyagent.slack.com/messages/CDTN970AX)
-  channel on [OPA Slack](https://slack.openpolicyagent.org/).
+- File [GitHub Issues](https://github.com/open-policy-agent/gatekeeper/issues)
+  to report bugs, request features, or ask questions asynchronously.
+  
```

Comment:
should we add https://github.com/open-policy-agent/community/discussions/categories/gatekeeper

Reply:
good idea